src/pages/index.astro

```diff
@@ -1,55 +1,97 @@
 ---
-import EmailForm from "../components/Email-Form.astro";
 import Layout from "../layouts/Layout.astro";
-import Hero from "../components/Hero.astro";
-import Features from "../components/Features.astro";
-import Notification from "../components/Notification.astro";
-import Footer from "../components/Footer.astro";
-import "../styles/index.css";
 ---
 
-<Layout title="Purple Bubble">
-  <main>
-    <div>
-      <script
-        type="text/javascript"
-        src="https://unpkg.com/@lottiefiles/lottie-player@1.6.0/dist/lottie-player.js"
-      ></script>
+<Layout title="Purple Bubble" fiscalSponsor="true">
+  <section>
+    <h2>Calling all (soon to be) Bubblers!</h2>
```

Comment:
the soon to be part feels a bit weird...

Reply:
fair

Reply:
@thepurplebubble/innerbubble does anyone have any different thoughts here?